src/deluge/gui/l10n/english.json

```diff
@@ -875,15 +875,18 @@
         "STRING_FOR_CLIP_NAME": "Clip Name",
         
         "STRING_FOR_NOTE_EDITOR": "Note Editor",
-        "STRING_FOR_NOTE_EDITOR_VELOCITY": "Velocity",
-        "STRING_FOR_NOTE_EDITOR_PROBABILITY": "Probability",
-        "STRING_FOR_NOTE_EDITOR_ITERANCE": "Iterance",
-        "STRING_FOR_NOTE_EDITOR_FILL": "Fill",
+        "STRING_FOR_NOTE_EDITOR_VELOCITY": "Note Velocity",
```

Comment:
Do we need these? I think it's fairly clear which velocity/probability you're editing from context, and strings eat into our ram (until someone figures out how to move them to external ram)

Reply:
I did this for when you're holding a note in the row editor and the display refreshes to show the value of that note (so it switches  from saying row probability to note probability).

If you don't think it's worth the extra strings I can remove them